recipes/r-xfun/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '0.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-xfun
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: xfun_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/xfun_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/xfun/xfun_{{ version }}.tar.gz
+  sha256: f226d4b603b04af3d1c5a8bbe9adfe06bae76aaf1ae1d04c3307f1c7eb0ba5ea
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('xfun')"  # [not win]
+    - "\"%R%\" -e \"library('xfun')\""  # [win]
+
+about:
+  home: https
```

Comment:
Could you please fix the URL? It should be https://github.com/yihui/xfun

The truncating of the home URL is a known bug in `conda skeleton cran` that has been fixed, but only in conda-build 3 (conda-forge is still working on migrating from conda-build 2).

Reply:
Fixed! Apologies for missing this.

Reply:
No worries @bsennblad! That's an easy one to miss.

Reply:
Thanks for approving.
It's a bit annoying that the ci Appveyor does not complete -- I don't think it took this long last time. Clicking on 'details' actually shows green light for the listed tests, which appear to have finished with success.
Do you think it is waiting for something more or has it just got stuck?